pkg/sense_hat_adapter.py

```diff
@@ -167,14 +179,22 @@ def update(self):
             self.set_cached_value(value)
             self.device.notify_property_changed(self)
 
-    def set_value(self, value):
-        if self.name == 'message':
-            self.device.controller.show_message(value)
-        elif self.name == 'color':
+    def set_value(prop, value):
```

Comment:
self is the object that method applies too, keep it or justify why it should be renamed